include/grl/kuka/KukaJAVAdriver.hpp

```diff
@@ -288,9 +289,27 @@ namespace grl { namespace robot { namespace arm {
 
           auto name = fbbP->CreateString(std::get<RobotName>(params_));
 
-          auto kukaiiwaArmConfiguration = flatbuffer::CreateKUKAiiwaArmConfiguration(*fbbP,name,commandInterface_,monitorInterface_);
+          auto clientCommandMode = grl::flatbuffer::EClientCommandMode_POSITION;
+          auto overlayType =  grl::flatbuffer::EOverlayType_NO_OVERLAY;
```

Comment:
These were not initialized/parsed before. The GRL_Driver.java does not use these currently if I am correct. I added these to avoid flatbuffer order change.